ChromeExtension/emoji-bar.html

```diff
@@ -8,7 +8,7 @@
 
     <style>
```

Comment:
I like the simplicity of the vertical emoji bar. Maybe just add color if you want to change the UI.